frontend/src/metabase/components/AccordionList.jsx

```diff
@@ -51,7 +49,9 @@ export default class AccordionList extends Component {
     className: PropTypes.string,
     id: PropTypes.string,
 
-    width: PropTypes.number,
+    // TODO: pass width to this component as solely number or string if possible
```

Comment:
Just as a general approach for future:
Do we want to immediately open related issues that are "tied" to these TODOs? There are far too many TODO comments in the code base so it's really easy to lose track of them and to forget them.

Reply:
That's actually a very good question. We can also mark those issues with a "good first issue" label, so if someone wants to contribute, it might be a good start

Reply:
@nemanjaglumac @kulyk 👍, I also like the idea of labeling these "Good first issue".

In this case I was investigating a bug, saw this, fixed it and filed the issue at the same time.

I think, with time, I'll be able to pinpoint and fix buxes more easily than now.
Until then, it's a lot of reading code.
It seems (to me) helpful to clean up obvious things around the issue, should make future changes gradually cheaper to implement.

Reply:
@gusaiani there is still a chance to open a follow up issue that can be labeled "Good first issue". The issue should be concerned about this TODO and the follow-up action needed around it.
That way we don't forget to do it.